docs/2.7.0/docs/canton/architecture/security.rst

```diff
@@ -141,9 +147,16 @@ The rotation frequency is fixed and cannot be changed.
 
    image taken from https://docs.aws.amazon.com/kms/latest/developerguide/rotate-keys.html
 
-If Canton is responsible for the storage of the private keys (i.e. we are using envelope encryption) the
-manual rotation of keys requires not only the creation of a new KMS key but also the re-encryption of our data with it.
-To do this Canton node administrators can request a :ref:`manual rotation of the KMS wrapper key through the Canton console <manual-aws-ksm-key-rotation>`.
+The manual rotation of a wrapper key requires not only the creation of a new KMS key but also the re-encryption of our data with it.
+To do this Canton node administrators can request a :ref:`manual rotation of the KMS wrapper key through the Canton console <manual-kms-wrapper-key-rotation>`.
+
+KMS Key Rotation
+^^^^^^^^^^^^^^^^
+
+When Canton's signing and encryption keys are off-sourced to a KMS (rather than encrypted at rest with a KMS wrapper key)
+their rotation has to be operated manually. Neither AWS or GCP provide automatic asymmetric key rotation.
+Nevertheless, this can be easily achieved by requesting a :ref:`standard rotation of Canton's keys <rotating-canton-keys>`,
```

Comment:
you should add the caveat that this only works when canton generates the new keys automatically, not when the new key has already been generated in the KMS (the GS custodian model). we need to document the latter too, can be in a separate PR if not already done.